flink-runtime/src/main/java/org/apache/flink/runtime/net/ConnectionUtils.java

```diff
@@ -352,7 +352,7 @@ private static boolean tryToConnect(
             return true;
         } catch (Exception ex) {
             String message =
-                    "Failed to connect from address '" + fromAddress + "': " + ex.getMessage();
+                    "Failed to connect from address '" + fromAddress + "' to address '" + toSocket + "': " + ex.getMessage();
```

Comment:
Could you deduplicate this line with 338? Extract
```
String detailedMessage = "to connect  to (toSocket) from local address fromAddres with timeout timeout"
```
(it's also a better message) and re-use `detailedMessage` both in L338 and L355?

Reply:
@pnowojski Thanks for your reminder, I have corrected it.